src/main/java/eu/dissco/core/digitalmediaobjectprocessor/security/WebSecurityConfig.java

```diff
@@ -1,45 +1,31 @@
 package eu.dissco.core.digitalmediaobjectprocessor.security;
 
-import org.keycloak.adapters.springsecurity.KeycloakSecurityComponents;
-import org.keycloak.adapters.springsecurity.authentication.KeycloakAuthenticationProvider;
-import org.keycloak.adapters.springsecurity.config.KeycloakWebSecurityConfigurerAdapter;
-import org.springframework.beans.factory.annotation.Autowired;
+import lombok.RequiredArgsConstructor;
+import org.springframework.boot.actuate.autoconfigure.security.servlet.EndpointRequest;
+import org.springframework.boot.actuate.health.HealthEndpoint;
 import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
-import org.springframework.security.core.authority.mapping.SimpleAuthorityMapper;
-import org.springframework.security.web.authentication.session.NullAuthenticatedSessionStrategy;
-import org.springframework.security.web.authentication.session.SessionAuthenticationStrategy;
+import org.springframework.security.config.http.SessionCreationPolicy;
+import org.springframework.security.web.SecurityFilterChain;
 
+@RequiredArgsConstructor
 @Configuration
 @EnableWebSecurity
-@ComponentScan(basePackageClasses = KeycloakSecurityComponents.class)
-public class WebSecurityConfig extends KeycloakWebSecurityConfigurerAdapter {
-
-  @Autowired
-  public void configureGlobal(AuthenticationManagerBuilder auth) {
-    KeycloakAuthenticationProvider keycloakAuthenticationProvider =
-        keycloakAuthenticationProvider();
-    SimpleAuthorityMapper grantedAuthorityMapper = new SimpleAuthorityMapper();
-    grantedAuthorityMapper.setConvertToUpperCase(true);
-    keycloakAuthenticationProvider.setGrantedAuthoritiesMapper(grantedAuthorityMapper);
-    auth.authenticationProvider(keycloakAuthenticationProvider);
-  }
-
+public class WebSecurityConfig {
+  private final JwtAuthConverter jwtAuthConverter;
   @Bean
-  @Override
-  protected SessionAuthenticationStrategy sessionAuthenticationStrategy() {
-    return new NullAuthenticatedSessionStrategy();
+  public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
+    http.authorizeHttpRequests()
+            .requestMatchers(EndpointRequest.to(HealthEndpoint.class)).permitAll()
```

Comment:
Can we also only expose the /health endpoint? Not sure what we are exposing this way, but looks more than we would like:
https://www.baeldung.com/spring-boot-actuators

Reply:
There are two steps to the actuator endpoints. 
1. Determine which endpoints are exposed
2. Determine which of the exposed endpoints need to be 

By default, only `/actuator/health` is exposed, so it's okay to put a blanket permission here. If we want to implement more actuator endpoints, we can revisit this. 

FYI You can verify what endpoints are exposed and what they reveal with the swager ui, just set the following in application.properties
`springdoc.show-actuator=true`